hr_attendance_warning/models/hr_employee.py

```diff
@@ -44,9 +44,9 @@ def _create_warning(self, w_type, date, min_int=False, max_int=False):
                 }
             )
         else:
-            self.env["hr.attendance.warning"].sudo(user=SUPERUSER_ID).create(
-                self._create_warning_vals(w_type, min_int, max_int)
-            )
+            self.env["hr.attendance.warning"].with_user(
```

Comment:
If you want to be admin, it is enough to do .sudo() Probably cleaner...

Reply:
Thank you :+1: